src/cgroup-network.c

```diff
@@ -472,8 +480,36 @@ void call_the_helper(const char *me, pid_t pid, const char *cgroup) {
     }
     else
         error("cannot execute cgroup-network helper script: %s", buffer);
+}
 
-    freez(m);
+int verify_path(const char *path) {
+    struct stat sb;
+
+    char c;
+    const char *s = path;
+    while((c = *s++)) {
+        if(c == '$' || c == '`') {
```

Comment:
If the goal is to prevent subshells in shell commands, it might be necessary to forbid '<' and '>' also: Bash supports `<(command)` and `>(command)`.